### ISSUE TITLE
[THoT] Remove filler scenarios and add extra character

### ISSUE THREAD

**Issue by nemaara** [wesnoth/wesnoth#4009]
Leaving as an issue for now because I'm not working on it quite yet. THoT has a lot of filler scenarios in the middle which add almost nothing to the campaign in terms of characterization or plot (they're really only bloat). 

I'm thinking about cutting it down to 7-8 scenarios (removing S4, 5, and 7, maybe S6 but not sure) and adding an extra character to the campaign. The extra character will be a masked dwarf who shows up in S1 after the orcs are dead in order to lure Aiglondur and Angarthing into Karrag's trap in Kal Kartha. 

Also, in the case that it happens, this will be helpful to transition to an eventual "half rewrite" like I did for TSG/DiD, but I'm not sure if I'm going to do that here.

**Comment by MetaKnight** [comment]
The reason Eric had that number of campaigns in the first place was so that he could strike a balance between army strength (number of veterans), gold and also because he felt the players would get bored of too many meaty scenarios. 
These scenarios exist to give the player a breath of fresh air- what sense would it make if each scenario was a beat x number of enemy leaders in a large battle. The whole prose and storyline of THoT was very well-written, with quality dialogue and reasonable progression.
I found the changes to 'Underlevels' appropriate, but this seems rather excessive. 
In short, the campaign is fine as it is. We should focus on other stuff that needs our attention, such as fixing up animations for certain units (Female Senior Village Elder, Bone Captain etc.), portraits for some units and other such stuff.

**Comment by shikadiqueen** [comment]
> In short, the campaign is fine as it is.
It really isn't.

Scenarios 3 through 7 are basically nothing but filler story-wise with just two elements (the relationship with NA-aligned orcs in S3 and the mage school in S7) serving a world-building function, and the latter is done in such a haphazard fashion (it's not alluded to anywhere outside the campaign) it might as well not exist.

In terms of gameplay, I'm sorry to be blunt but ESR hasn't ever been the best judge of that. That "breath of fresh air" essentially turned the campaign into a long series of low-difficulty scenarios with uninteresting fights that only increase the overall playtime without anything to show for it. In many cases the scenarios feel like playing against the AI on mainline MP maps. Again, scenario 8 seems to be the turning point at which the author finally decides to get off his seat and make things happen.

Overall it's a campaign that has a lot of unexploited potential due to its poor execution and the author's obsession with playing by his own rulebook. Wesnoth can and should do much better than that if it ever wants to stop being stuck in its self-inflicted "player-generated content is better than developer-maintained content" limbo.

> We should focus on other stuff that needs our attention, such as fixing up animations for certain units (Female Senior Village Elder, Bone Captain etc.), portraits for some units and other such stuff.

By all means, please do work on that stuff if it strikes your fancy. It's just as important as making mainline content better able to showcase the game's potential as a creative platform.
